### PR TITLE
Fix PXC_QA framework to work with Pstress instead of Pquery

### PR DESCRIPTION
**Pull request by kaushikpuneet07**
Fixed PXC_QA framework to work with Pstress instead of Pquery. Pquery configs has been replaced by pstress config.

### CODE REVIEW

File: suite/random_qa/pstress_crash_recovery_qa.py [hunk 1]
```diff
@@ -80,19 +80,19 @@ def startup_check(self, cluster_node):
...
-            PQUERY_CMD = PQUERY_BIN + " --database=" + db + " --threads=50 --logdir=" + \
+            PSTRESS_CMD = PSTRESS_BIN + " --database=" + db + " --threads=50 --logdir=" + \
                          WORKDIR + "/log --log-all-queries --log-failed-queries --user=root --socket=" + \
-                         socket + " --seed 1000 --tables 25 --records 1000 " + \
-                         PQUERY_EXTRA + " --seconds 300 --undo-tbs-count 0  --sql-file " + \
-                         PQUERY_GRAMMER_FILE + " --step " + str(i) + " > " + \
-                         WORKDIR + "/log/pquery_run.log"
-            utility_cmd.check_testcase(0, "PQUERY RUN command : " + PQUERY_CMD)
-            query_status = os.system(PQUERY_CMD)
+                         socket + " --seed 1000 --tables 5 --records 100 " + \
```
Comment (mohitj1988) [inline suite/random_qa/pstress_crash_recovery_qa.py]: IMO, we should change hard coded seed value, instead we should create a function that generates random numbers eg. --seed $RANDOMN
or else, PXC-QA framework will create exact same data.

File: suite/random_qa/pstress_crash_recovery_qa.py [hunk 2]
```diff
@@ -80,19 +80,19 @@ def startup_check(self, cluster_node):
...
+                         PSTRESS_EXTRA + " --seconds 30 --grammar-file " + \
```
Comment (mohitj1988) [inline suite/random_qa/pstress_crash_recovery_qa.py]: Please increase the default test duration to 60sec

File: suite/random_qa/pstress_crash_recovery_qa.py [hunk 3]
```diff
@@ -80,19 +80,19 @@ def startup_check(self, cluster_node):
...
+                         PSTRESS_GRAMMAR_FILE + " --step " + str(i) + " > " + \
+                         WORKDIR + "/log/pstress_run.log"
+            utility_cmd.check_testcase(0, "PSTRESS RUN command : " + PSTRESS_CMD)
+            query_status = os.system(PSTRESS_CMD)
             if int(query_status) != 0:
-                utility_cmd.check_testcase(1, "ERROR!: PQUERY run is failed")
+                utility_cmd.check_testcase(1, "ERROR!: PSTRESS run is failed")
```
Comment (mohitj1988) [inline suite/random_qa/pstress_crash_recovery_qa.py]: Maybe write: PSTRESS run failed. Remove "is"

File: suite/random_qa/pstress_random_qa.py [hunk 4]
```diff
@@ -20,10 +20,10 @@
 args = parser.parse_args()
 if args.encryption_run is True:
     encryption = 'YES'
-    PQUERY_EXTRA = ""
+    PSTRESS_EXTRA = ""
 else:
     encryption = 'NO'
-    PQUERY_EXTRA = "--no-enc"
+    PSTRESS_EXTRA = "--no-encryption"
```
Comment (mohitj1988) [inline suite/random_qa/pstress_random_qa.py:26]: Please evaluate why we need --no-encryption. PXC-8.0 supports encryption run

File: suite/random_qa/pstress_crash_recovery_qa.py (hunk 1 above)
Comment (kaushikpuneet07) [reply suite/random_qa/pstress_crash_recovery_qa.py]: Fixed

File: suite/random_qa/pstress_crash_recovery_qa.py (hunk 2 above)
Comment (kaushikpuneet07) [reply suite/random_qa/pstress_crash_recovery_qa.py]: fixed

File: suite/random_qa/pstress_crash_recovery_qa.py (hunk 3 above)
Comment (kaushikpuneet07) [reply suite/random_qa/pstress_crash_recovery_qa.py]: fixed

File: pstress/pstress-run.ini [hunk 5]
```diff
@@ -12,8 +12,8 @@ encryption = yes
 myextra = ''
 
 [pquery]
```
Comment (mohitj1988) [inline pstress/pstress-run.ini]: This should also be changed to pstress

**Comment by mohitj1988** [comment]
Please remove references of pquery from 
README.md
randgen/util/reducer/reducer.sh

File: suite/random_qa/pstress_crash_recovery_qa.py [hunk 6]
```diff
@@ -80,19 +81,20 @@ def startup_check(self, cluster_node):
...
-            PQUERY_CMD = PQUERY_BIN + " --database=" + db + " --threads=50 --logdir=" + \
+            PSTRESS_CMD = PSTRESS_BIN + " --database=" + db + " --threads=50 --logdir=" + \
                          WORKDIR + "/log --log-all-queries --log-failed-queries --user=root --socket=" + \
-                         socket + " --seed 1000 --tables 25 --records 1000 " + \
-                         PQUERY_EXTRA + " --seconds 300 --undo-tbs-count 0  --sql-file " + \
-                         PQUERY_GRAMMER_FILE + " --step " + str(i) + " > " + \
-                         WORKDIR + "/log/pquery_run.log"
-            utility_cmd.check_testcase(0, "PQUERY RUN command : " + PQUERY_CMD)
-            query_status = os.system(PQUERY_CMD)
+                         socket + " --seed " + str(n) + " --tables 5 --records 100 " + \
```
Comment (mohitj1988) [inline suite/random_qa/pstress_crash_recovery_qa.py]: I think we should not change the value of table and number of records.
5 tables and 100 records are too less.

Please restore the original value tables 25, records 1000, seconds 300

File: pstress/pstress-run.ini (hunk 5 above)
Comment (kaushikpuneet07) [reply pstress/pstress-run.ini]: Fixed

File: suite/random_qa/pstress_crash_recovery_qa.py (hunk 6 above)
Comment (kaushikpuneet07) [reply suite/random_qa/pstress_crash_recovery_qa.py]: Fixed , updated to 15 tables and 300 records

Comment (mohitj1988) [reply suite/random_qa/pstress_crash_recovery_qa.py]: Why not restored original value? pstress is capable of generating upto 100+ tables in 300 seconds. Don't you want PXC QA framework to find more bugs ;) ??

**Comment by mohitj1988** [comment]
Rest all Looks good to me. Please fix the last comment and then merge
Dont forget to Squash and merge and add JIRA ticket link in Description.

File: suite/random_qa/pstress_crash_recovery_qa.py (hunk 6 above)
Comment (kaushikpuneet07) [reply suite/random_qa/pstress_crash_recovery_qa.py]: restored to original value

**Pull request merged**